### DISCUSSION TITLE
[Bug] working directory is not changed / hydra.job.chdir is null

### DISCUSSION THREAD

**Discussion by raphCode** [facebookresearch/hydra#2283] [Q&A]
# 🐛 Bug
## Description
Despite the [documentation saying hydra changes the working directory](https://hydra.cc/docs/tutorials/basic/running_your_app/working_directory/), it does not. 

## To reproduce

Run the documentation example:
https://github.com/facebookresearch/hydra/blob/be5b345f370fb18d8db24f676b638e2603779102/examples/tutorials/basic/running_your_hydra_app/3_working_directory/my_app.py

I even ran it with `--cfg hydra` and saw that `hydra.job.chdir` is null.

## Expected Behavior

Hydra changes the app's working directory before calling the function decorated with `hydra.main()`.

## System information
- **Hydra Version** :  Hydra 1.2.0
- **Python version** : Python 3.10.5
- **Virtual environment type and version** : venv
- **Operating system** : Arch Linux

**Comment by jieru-hu** [comment]
hi @raphCode pls check this upgrade guide https://hydra.cc/docs/upgrades/1.1_to_1.2/changes_to_job_working_dir/

**Reply by raphCode** [reply]
Hi, thanks for the directions.

Is this a documentation issue then, because the section I linked reads like 1.2 should change the working dir automatically?
Also the documentation example does not work like expected.

**Comment by Queuecumber** [comment]
I'd like to second that this is a documentation issue, official 1.2 docs state:

```
By default, Hydra's @hydra.main decorator makes a call to os.chdir before passing control to the user's decorated main function. Set hydra.job.chdir=False to disable this behavior.
```

https://hydra.cc/docs/1.2/tutorials/basic/running_your_app/working_directory/#disable-changing-current-working-dir-to-jobs-output-dir

**Comment by Queuecumber** [comment]
This is still wrong in the documentation https://hydra.cc/docs/1.3/tutorials/basic/running_your_app/working_directory/#disable-changing-current-working-dir-to-jobs-output-dir

**Reply by Jasha10** [reply]
Thanks very much @Queuecumber; I've just opened PR https://github.com/facebookresearch/hydra/pull/2643 to update the docs.
